node/src/BaseClient.ts

```diff
@@ -1126,12 +1131,58 @@ export class BaseClient {
      */
     public zcount(
         key: string,
-        minScore: ScoreLimit,
-        maxScore: ScoreLimit,
+        minScore: ScoreLimit<number>,
+        maxScore: ScoreLimit<number>,
```

Comment:
Please follow python names -> change ScoreLimit to ScoreBoundary, since LIMIT is a parameter for the sorted set commands so it can be confusing

Reply:
Since ScoreBoundary is being used both for score and lex, please see if you can find a name that represents them both